front-end/src/Pages/_PROTECTED/User/Dashboard/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react';
+import './dashboard.css';
+import Toast from 'App/Swal2/toast';
+import { useCookies } from 'react-cookie';
+import { logout } from 'API/userAPI';
+import { apiCall } from 'API/index';
+
+export default function Dashboard() {
+  const [cookie, , removeCookie] = useCookies();
+
+  useEffect(() => {
+    apiCall.defaults.headers.Authorization = `Bearer ${cookie.token}`;
+  }, []);
```

Comment:
Why you still need to set again the authorization here and in admin? You already set it after login success.